library/bind.c

```diff
@@ -518,9 +524,15 @@ static void stop_binding(struct binding_s *b) {
                     .length = sizeof(conn_id),
                     .value = (uint8_t *) &conn_id
             },
+            {
+                    .header_id = ListenerId,
+                    .length = sizeof(b->conn->server.listener_id),
+                    .value = (uint8_t*)b->conn->server.listener_id,
```

Comment:
we're sending raw bytes over the wire. should the value be normalized for different endian in some way?

Reply:
the bites are not interpreted is any way and are used for comparisons only